pulsar-io/debezium/mysql/src/main/resources/debezium-mysql-source-config.yaml

```diff
@@ -19,17 +19,13 @@
 
 tenant: "test"
 namespace: "test-namespace"
-name: "debezium-kafka-source"
-topicName: "kafka-connect-topic"
-archive: "connectors/pulsar-io-kafka-connect-adaptor-2.3.0-SNAPSHOT.nar"
+name: "debezium-mysql-source"
+topicName: "debezium-mysql-topic"
+archive: "connectors/pulsar-io-debezium-mysql-2.4.0-SNAPSHOT.nar"
```

Comment:
do we need this archive? `debezium-mysql` is a builtin connector, no?

Reply:
If we use local run, we should give the archive.